main.py

```diff
@@ -0,0 +1,167 @@
+import json
+import datetime
+
+isAlarm = True
```

Comment:
```suggestion
is_alarm = True
```

파이썬에서는 `snake_case` 컨벤션을 사용합니당